sql/src/test/java/io/crate/integrationtests/JoinIntegrationTest.java

```diff
@@ -320,9 +320,13 @@ public void testCrossJoinFromInformationSchemaTable() throws Exception {
     public void testSelfJoin() throws Exception {
         execute("create table t (x int)");
         ensureYellow();
-        execute("insert into t (x) values (1)");
+        execute("insert into t (x) values (1), (2)");
         execute("refresh table t");
-        execute("select * from t as t1, t as t2");
+        execute("select * from t as t1, t as t2 order by t1.x");
+        assertThat(printedTable(response.rows()), is("1| 1\n" +
```

Comment:
Because with 1 row we have the same value in left and right in NL and hides any potential issue there